index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <header id="header">
+      <img src="https://www.google.com/search?q=image+of+headphone&rlz=1CAZAQH_enNG975&sxsrf=APq-WBuwS8F0KaP4AmHrYaDb2sSPQnEl8w:1648298707986&tbm=isch&source=iu&ictx=1&vet=1&fir=mIT4NCgfDXtE1M%252Ch5tqX_QHG5wDXM%252C_%253BgsCGxq3lpLsMWM%252Ch5tqX_QHG5wDXM%252C_%253B5BNEhzqLR8Dc3M%252Ch5tqX_QHG5wDXM%252C_%253BSqta8j3U9KnYJM%252C-wyP6cFsRWXx7M%252C_%253BVGj7RF0Q6tYZ6M%252C-wyP6cFsRWXx7M%252C_%253B7Hn9sHMC2jC5-M%252CG1miClKgJgnM-M%252C_%253B5L7yigwMDOgX0M%252Ch5tqX_QHG5wDXM%252C_%253BcHHelTGnHkyADM%252CG1miClKgJgnM-M%252C_%253B0H7OSFnnaPMuIM%252CG1miClKgJgnM-M%252C_%253Bwb1KJQBRuykx4M%252C-wyP6cFsRWXx7M%252C_%253BhHVNOMYfMCE-cM%252Ch5tqX_QHG5wDXM%252C_%253B3-xY1ayV8GzawM%252CuvtEsPaVDI4FUM%252C_%253BmmIGoVH3_psyrM%252CeMa4xi-J2781NM%252C_%253BnMt1CHAO49p7FM%252CY7LLjHEJRvCexM%252C_&usg=AI4_-kTzMrwndBrh_DT1ycL_xHAt6KEtTw&sa=X&ved=2ahUKEwi76-OX5-P2AhUihf0HHeHsCX8Q9QF6BAgpEAE#imgrc=gsCGxq3lpLsMWM" id="header-img">
+      <nav id="nav-bar">
+          <a href="#header" class="nav-link">About</a>
+          <a href="#header" class="nav-link">Contact us</a>
+          <a href="#header" class="nav-link">More products</a>
+      </nav>
+    </header>
+    <video src="https://www.youtube.com/watch?v=sBba4MGPc2U" id="video"></video>
+    <form action="https://www.freecodecamp.com/email-submit" id="form">
+      <input name="email" type="email" id="email" placeholder="Input your email here">
+      <button id="submit"><a href="https://www.freecodecamp.com/email-submit"> </a></button>
```

Comment:
Please try defining an input with a ' type="button" ' attribute. It helps browsers in identifying the functionality of the tag